src/CMakeLists.txt

```diff
@@ -41,6 +41,10 @@ add_executable(${executable_name} MACOSX_BUNDLE ${multivnc_SRCS})
 
 target_link_libraries(${executable_name} MultiVNCgui ${wxWidgets_LIBRARIES} wxservdisc vncclient)
 
+if(APPLE)
```

Comment:
this should go into the above `if (APPLE)` stanza and be surrounded by `if (wxWidgets_VERSION_STRING LESS 3.1)`

Reply:
It doesn't work that way:
```
CMake Error at src/CMakeLists.txt:38 (set_target_properties):
  set_target_properties Can not find target to add properties to: MultiVNC
```
I guess it doesn't know about the target yet. Have to be after `add_executable`.